config/locales/en/course/discussion/post.yml

```diff
@@ -0,0 +1,5 @@
+en:
+  course:
+    discussion:
+      post:
+        confirm-delete-post: 'Are you sure that you want to delete this post?'
```

Comment:
Let's pick one and go with it, we can always change wording later. Maybe we just use @kxmbrian's current one?